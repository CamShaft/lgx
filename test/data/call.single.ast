#{
  type => call,
  line => 1,
  value => {module, function},
  children => [
    #{
      type => literal,
      value => <<"binary">>,
      line => 2
    },
    #{
      type => literal,
      value => 123,
      line => 3
    },
    #{
      type => literal,
      value => 3.14,
      line => 4
    }
  ]
}.