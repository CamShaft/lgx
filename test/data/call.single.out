{module, function, [<<"binary">>, 123, 3.14]}.